.github/workflows/main.yml

```diff
@@ -5,17 +5,28 @@ on:
   # Triggers the workflow on pull request events and merges/pushes to main
   push:
     branches:
-      - '**'
+      - main
+      - release/*
```

Comment:
Does this change when builds are triggered? If I push to a branch with a PR open will that trigger a build?

Reply:
Yeah this does change it.

Previously on ** it would run on any push to any branch. Now it will only run on pushes to main or release/*

workflow_dispatch below means you can trigger it manually for any branch if you feel like it

Reply:
Yep! Pushing to a branch with a PR will still result in a build since we're also triggering on `synchronize` for PRs.

Changing this to `main` stops GitHub Actions from creating two identical builds when pushing to pull requests:

Example:
<img width="1303" alt="Screenshot 2023-06-21 at 1 39 33 pm" src="https://github.com/OctopusDeploy/Versioning/assets/25342760/0f184568-b227-408f-a4d7-c37fe898a8df">

Reply:
Sweet, sounds good 👍